corehq/apps/fixtures/fixturegenerators.py

```diff
@@ -6,6 +6,22 @@
 from dimagi.utils.modules import to_function
 
 
+def item_lists_by_domain(domain):
+    ret = list()
+    for data_type in FixtureDataType.by_domain(domain):
+        ret.append({
+            'sourceUri': 'jr://fixture/item-list:%s' % data_type.tag,
+            'defaultId': data_type.tag,
+            'initialQuery': "instance('{tag}')/item-list:{tag}/{tag}".format(tag=data_type.tag),
+            'name': data_type.tag,
+            'structure': {
+                f.field_name: {
+                    'name': f.field_name,
+                    'no_option': True
```

Comment:
What's the point of sending these if they have no children and `no_option` is true?